### DISCUSSION TITLE
Lesson 6 Operation timeout

### DISCUSSION THREAD

**Discussion by dodeyeikoi** [Cyfrin/foundry-full-course-cu#845] [General]
Hey guys, I just started lesson 6,  I'm trying to install the wsl on my windows 11 but this is the error I get:

```
Copyright (C) Microsoft Corporation. All rights reserved.

Install the latest PowerShell for new features and improvements! https://aka.ms/PSWindows

PS C:\Users\User> wsl --install
The operation timed out

```

how can I solve this please ?

**Comment by capsteval** [comment]
> Hey guys, I just started lesson 6, I'm trying to install the wsl on my windows 11 but this is the error I get:
> 
> ```
> Copyright (C) Microsoft Corporation. All rights reserved.
> 
> Install the latest PowerShell for new features and improvements! https://aka.ms/PSWindows
> 
> PS C:\Users\User> wsl --install
> The operation timed out
> ```
> 
> how can I solve this please ?

I'm experiencing the same thing. how were you able to get past here?

**Comment by capsteval** [comment]
i was able to solve this by following the manual wsl installation and installing wsl2 worked for me.
use this microsoft documentation
https://learn.microsoft.com/en-us/windows/wsl/install-manual

ensure you enable windows feature: windows subsystem for linux 
install latest ubuntu and 
you should be good.

**Reply by EngrPips** [reply]
Awesome, Thanks for the heads-up buddy.
